### DISCUSSION TITLE
Version 2.1.3

### DISCUSSION THREAD

**Discussion by nimrod-gileadi** [google-deepmind/mujoco#199] [Announcements]
This is a minor release, handling issues discovered after the launch of 2.1.2.

## General

1. `simulate` now support cycling through cameras (with `[` and `]` keys).
2. `mjVIS_STATIC` toggles all static bodies, not just direct children of the world.

## Python bindings

3. Added a `free()` method to `MjrContext`, rather than relying on the object destructor.
4. Enums now support arithmetic and bitwise operations with numbers.

## Bug fixes

5. Fixed a rendering bug for planes, introduced in 2.1.2. This broke maze environments in [`dm_control`](https://github.com/deepmind/dm_control).

<hr /><em>This discussion was created from the release <a href='https://github.com/deepmind/mujoco/releases/tag/2.1.3'>2.1.3</a>.</em>

**Comment by peterdavidfagan** [comment]
Thanks for the work that goes into maintaining MuJoCo. The python bindings and further support in recent releases has been awesome.
